cvmfs/cvmfs.cc

```diff
@@ -1270,6 +1270,41 @@ static void cvmfs_statfs(fuse_req_t req, fuse_ino_t ino) {
   fuse_reply_statfs(req, &info);
 }
 
+const uint64_t kMaxMetainfoLength = 64*1024;
+
+string GetRepoMetainfo(uint64_t size) {
+  if (!mount_point_->catalog_mgr()->manifest()) {
+    return "Manifest not available";
```

Comment:
This string and the other error strings need to be truncated if `size` is smaller than their length